### PR TITLE
Contrast 37799

### PR DESCRIPTION
**Pull request by kaitlynmortimer**
These changes will be merged as soon as the transition by CS is complete.

### CODE REVIEW

File: content/troubleshooting/net/issues/AnotherIISProfiler.md [hunk 1]
```diff
@@ -17,3 +17,5 @@ The CLR Profiling API only allows a single agent to be attached to a .NET proces
 
 To resolve the issue, Contrast implemented profiler chaining that allows Contrast to run side-by-side with another profiling tool. To enable profiling, see the `ProfilerChainingEnabled` configuration setting](installation-netconfig.html#overview).
 
+<!-- Should we update this with the equivalent common config property?  -->
+
```
Comment (kaitlynmortimer) [inline content/troubleshooting/net/issues/AnotherIISProfiler.md]: @DanFiedler See commented text above

File: content/installation/net/install/System-requirements-net.md [hunk 2]
```diff
@@ -31,3 +31,6 @@ Before installing the .NET agent, confirm that you can meet the following requir
 > * The .NET agent roughly doubles the RAM requirements of analyzed applications. Applications should use less than half of the available memory when the .NET agent is not installed. 
 >
 > * The .NET agent uses the CLR Profiling API to perform data and code flow analysis - detect SQL-injection, XSS, weak cryptography, etc. - as well as to detect libraries and technologies used by analyzed applications. The Contrast agent can now exist alongside other .NET Profiler agents, such as performance or APM tools with the `agent.dotnet.enable_chaining` [configuration setting](installation-netconfig.html#overview) enabled.
+
+<!-- Replace `agent.dotnet.enable_chaining` property?  -->
+
```
Comment (kaitlynmortimer) [inline content/installation/net/install/System-requirements-net.md]: @DanFiedler See commented text above

Comment (DanFiedler) [reply content/installation/net/install/System-requirements-net.md]: No, that's the correct config.

File: content/troubleshooting/net/issues/AnotherIISProfiler.md (hunk 1 above)
Comment (DanFiedler) [reply content/troubleshooting/net/issues/AnotherIISProfiler.md]: Yes we should replace that legacy configuration with agent.dotnet.enable_chaining

**Pull request merged**